src/styles/main.scss

```diff
@@ -1,3 +1,258 @@
-body {
+$htmlFont: "Roboto", sans-serif;
+$htmlFont-weight: 500;
+$htmlFont-size: 12px;
+$color-text: #060b35;
+$color-text-grey: #616070;
+$color-background: #fff;
+$color-accent-blue: #00acdc;
+$width-card: 200px;
+$gap-card: 50px;
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: $htmlFont;
+  font-weight: $htmlFont-weight;
+  font-size: $htmlFont-size;
+}
+
+body,
+h2,
+h3 {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  text-align: center;
+}
+
+.nav {
+  margin-right: 50px;
+
+  &__list {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    margin: 0;
+    padding: 0;
+    list-style: none;
+    text-transform: uppercase;
+  }
+
+  &__item:not(:first-child) {
+    margin-left: 20px;
+  }
+
+  &__link {
+    display: block;
+    text-decoration: none;
+    line-height: 60px;
+    color: $color-text;
+  }
+
+  &:hover {
+    color: $color-accent-blue;
+  }
+}
+
+.logo_img {
+  display: flex;
+  justify-content: flex-start;
+  margin-left: 50px;
+  margin-top: 10px;
+}
+
+.is-active {
+  position: relative;
+  color: $color-accent-blue;
+
+  &::after {
+    position: absolute;
+    content: "";
+    width: 100%;
+    background-color: $color-accent-blue;
+    height: 4px;
+    left: 0;
+    bottom: 0;
+    border-radius: 8px;
+  }
+
+}
+
+.main {
+  display: flex;
+  justify-content: center;
+}
+
+.card {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  flex-wrap: wrap;
+  width: (4 * $width-card) + (3 * $gap-card);
+  margin-top: 51px;
+  margin-bottom: 73px;
+}
+
+.card__card {
+  display: flex;
+  flex-direction: column;
+  width: $width-card;
+  margin-right: $gap-card;
```

Comment:
you have set the `margin-right` for each cards that's why the width for parent block won't work correctly